ledger/apply/payment_test.go

```diff
@@ -221,6 +221,11 @@ func TestPaymentValidation(t *testing.T) {
 		if badFee.WellFormed(spec, tc.Proto) == nil {
 			t.Errorf("transaction with no fee %#v verified incorrectly", badFee)
 		}
+		badFee.Fee.Raw = 1
+		if badFee.WellFormed(spec, tc.Proto) == nil {
+			t.Errorf("transaction with low fee %#v verified incorrectly", badFee)
+		}
```

Comment:
I'm confused what this checks. `WellFormed` seems to confirm that individual transactions are well formed but this PR's implementation deals specifically with group transactions. Wouldn't a unit test for this PR rather test that transaction group fees are correct, as opposed to individual transactions?

Reply:
Good point.  I think this unit test should have existed before.  I wrote it while I was still feeling my way around, before I wrote any of this code.  It will, in fact, never be triggered after the protocol upgrade that this code requires.

But it does give me something to test against when writing the integration tests.  I wanted to confirm that a low fee transaction *was* bad, but now is not.